### ISSUE TITLE
[CAIA Intake] 10-10 Health Apps team - 10-10EZ TERA (Toxic Exposure)

### ISSUE THREAD

**Issue by hdjustice** [department-of-veterans-affairs/va.gov-team#74254]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: 10-10 Health Apps
- **OCTO product owner:** Patrick Bateman
- **Product name:** 10-10EZ
- **Product manager:**  Heather Justice
- **Slack channel:**  [#1010-health-apps](https://dsva.slack.com/archives/CMJ2V70UV)
- **Dedicated content writer on your team (if you have one):** Jessica Stump (@jessicastump) and Angela Agosto (@aagosto90)
- **Dedicated a11y specialist on your team (if you have one):** N/A
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

We need to remove some items from the Veteran's "Additional Information" page, and add some questions regarding Toxic Exposure (TERA)
TERA = Toxic Exposure Risk Activity

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [x] Yes
- [ ] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

We are currently in the wireframing/design stage.
We have a target launch date of 3/5/2024

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [x] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** N/A

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [TERA Initiative Brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/application/va-application/Toxic%20Exposure/Toxic%20Exposure%20(TERA)%20questions%20-%20Initiative%20Brief.md)
- [Latest draft of 10-10EZ with TERA questions (1/23/2024)](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/application/va-application/Toxic%20Exposure/TERA%20draft%201-23-2024%20VA%20Form%2010-10EZ.pdf)

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [x] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA operations
- [x] Initial Review - @coforma-terry and @strelichl
```

```[tasklist]
### CAIA a11y tasks
```

```[tasklist]
### CAIA content tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/75267
```

```[tasklist]
### CAIA IA tasks
```

**Comment by aliyahblackmore** [comment]
**Additional links:** 

[Mural board](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1699473497914/86412743291c286587ff315e52549e98e06ba534?fromVisitorModal=true&sender=u5ad49c107baa41137f271007)
[Figma page](https://www.figma.com/file/UljiHam46o5DItC5iDgmPd/10-10EZ?type=design&node-id=450-19244&mode=design&t=GyO36RKAPEFgXEna-0)
[Sitewide PACT wizard](https://staging.va.gov/pact-act-eligibility/service-period-1) tool for reference

**Comment by aliyahblackmore** [comment]
**Some updates here:**

- 1010 team is waiting on ES.
- In the mean time, we can start to look at this content and make adjustments.
- I have a question for Danielle on how we refer to the toxic exposure questions - I added this as an agenda item for our Monday sync.
- Their team has a meeting with the 526 on their toxic exposure design.

**Comment by hdjustice** [comment]
TERA questions launched March 5th.
